web/src/emoji_picker.ts

```diff
@@ -345,15 +371,20 @@ function update_emoji_showcase($focused_emoji) {
     $(".emoji-showcase-container").html(rendered_showcase);
 }
 
-function maybe_change_focused_emoji($emoji_map, next_section, next_index, preserve_scroll) {
+function maybe_change_focused_emoji(
+    $emoji_map: JQuery,
+    next_section: number,
+    next_index: number,
+    preserve_scroll?: boolean,
+): boolean {
     const $next_emoji = get_rendered_emoji(next_section, next_index);
     if ($next_emoji) {
         current_section = next_section;
         current_index = next_index;
         if (!preserve_scroll) {
             $next_emoji.trigger("focus");
         } else {
-            const start = scroll_util.get_scroll_element($emoji_map).scrollTop();
+            const start = scroll_util.get_scroll_element($emoji_map).scrollTop() ?? 0;
```

Comment:
I think this can be `assert`ed  ; Given that we have `$emoji_map`. See the example where we have used minimalistic asserts.

Reply:
I think this applies to a lot of `??` here for ex. `next_index` etc.